### PR TITLE
feat: Continual learning use case

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/retail-use-cases/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Add Continual learning reference demo

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

File: use-cases/continual_learning/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Continual Learning Demo
+
+## Description
+
+This software package delivers the building blocks required to develop an AI-powered weighing scale use case that continuously learns and refines its image classification model with incoming new data. This reference implementation provides an intuitive application that facilitates an image classification model to learn incrementally with new image samples (sourced either from local images folder or live streaming feed). The continual learning algorithm is based on Streaming Linear Discriminant Analysis (SLDA) [[1]](#references) enabling the model to learn from individual images without forgetting previously acquired knowledge, resulting in faster training performance with comparable accuracy (Top1/Top5). This solution is optimally suited for CPU utilization, ensuring its adaptability across diverse hardware setups. This software has been tested to work with Python 3.8.10.
```
Comment (jim-wang-intel) [inline use-cases/continual_learning/README.md:5]: Is Python 3.8.10 the only working version? maybe re-work this a bit more like 3.8.10+ if that is working in general

File: use-cases/continual_learning/camera-demo.py [hunk 2]
```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/env python
```
Comment (jim-wang-intel) [inline use-cases/continual_learning/camera-demo.py:1]: missing copyright header

File: use-cases/continual_learning/notebook/slda_frozen_classification_cl.ipynb [hunk 3]
```diff
@@ -0,0 +1,526 @@
+{
```
Comment (NeethuES-intel) [inline use-cases/continual_learning/notebook/slda_frozen_classification_cl.ipynb:1]: doesn't this file need copyright ?

File: use-cases/continual_learning/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+tqdm==4.66.3
```
Comment (NeethuES-intel) [inline use-cases/continual_learning/requirements.txt:1]: copyright ?

**Review by jim-wang-intel — APPROVED** [review]

**Review by NeethuES-intel — APPROVED** [review]

**Pull request merged**